src/edit.js

```diff
@@ -277,6 +335,66 @@ export default function Edit({attributes, setAttributes, instanceId}) {
                                 amountChanged={(amounts) => setAttributes({donationAmounts: amounts})}
                             />
                         </PanelRow>
+                        <PanelRow>
+                            <div className={'dfb-recaptcha-options-wrap'}>
+                                <ToggleControl
+                                    label={__('Enable Google ReCAPTCHA', 'donation-form-block')}
+                                    help={
+                                        <>
+                                            {__(
+                                                'Many forms of fraud, including card testing, can be prevented by using ReCAPTCHA. This is a free service provided by Google.',
```

Comment:
```suggestion
                                                'Many forms of fraud, including card testing, can be prevented by using ReCAPTCHA, a free service provided by Google. ',
```
(there needs to be a space at the end.)